### PR TITLE
Improvement of test coverage of datasafe

### LINKED ISSUE #245: Improvement of test coverage of the datasafe component
Test coverage report

Improve test coverage to 100%

### PR DESCRIPTION
**Pull request by Thendo20**

### CODE REVIEW

File: datasafe-business/pom.xml [hunk 1]
```diff
@@ -184,6 +184,12 @@
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>2.15.4</version>
```
Comment (max402) [inline datasafe-business/pom.xml]: why version 2.15.4. Try to use ${jackson.version} variable. It's defined in parent pom.

File: datasafe-simple-adapter/datasafe-simple-adapter-impl/pom.xml [hunk 2]
```diff
@@ -73,6 +73,12 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (max402) [inline datasafe-simple-adapter/datasafe-simple-adapter-impl/pom.xml]: Do you need to change something in datasafe-simple-adapter at all?

Comment (Thendo20) [reply datasafe-simple-adapter/datasafe-simple-adapter-impl/pom.xml]: No, I am going to remove my changes from datasafe-simple-adapter

File: datasafe-directory/datasafe-directory-impl/src/test/java/de/adorsys/datasafe/directory/impl/profile/dfs/BucketAccessServiceImplTest.java [hunk 3]
```diff
@@ -14,6 +15,7 @@
 class BucketAccessServiceImplTest extends BaseMockitoTest {
 
     private static final String ABSOLUTE_BUCKET = "s3://bucket";
+    private final Uri uri = new Uri(ABSOLUTE_BUCKET);
```
Comment (max402) [inline datasafe-directory/datasafe-directory-impl/src/test/java/de/adorsys/datasafe/directory/impl/profile/dfs/BucketAccessServiceImplTest.java]: no usages of uri

File: datasafe-encryption/datasafe-encryption-api/src/main/java/de/adorsys/datasafe/encrypiton/api/types/encryption/MutableEncryptionConfig.java [hunk 4]
```diff
@@ -200,9 +200,9 @@ KeyCreationConfig.EncryptingKeyCreationCfg toEncryptingKeyCreationCfg() {
                     builder.sigAlgo(sigAlgo);
                 }
 
-                if (null != curve) {
+
                     builder.curve(curve);
-                }
```
Comment (max402) [inline datasafe-encryption/datasafe-encryption-api/src/main/java/de/adorsys/datasafe/encrypiton/api/types/encryption/MutableEncryptionConfig.java:205]: tests work also without this change

File: datasafe-encryption/datasafe-encryption-impl/src/test/java/de/adorsys/datasafe/encrypiton/impl/cmsencryption/CmsEncryptionServiceImplTest.java [hunk 5]
```diff
@@ -28,17 +33,10 @@
...
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.RandomAccessFile;
+import java.io.*;
```
Comment (max402) [inline datasafe-encryption/datasafe-encryption-impl/src/test/java/de/adorsys/datasafe/encrypiton/impl/cmsencryption/CmsEncryptionServiceImplTest.java:36]: change in Intellij settings to not automatically replace import with *
Preferences - Editor - Code Style - Java - Imports - Class count to use import with '*' - set 999

**Review by max402 — APPROVED** [review]

**Pull request merged**